apps/modernization-ui/src/apps/page-builder/pages/AddNewPage/AddNewPageFields.tsx

```diff
@@ -150,20 +150,24 @@ export const AddNewPageFields = (props: AddNewPageFieldProps) => {
             <Controller
                 control={form.control}
                 name="pageDescription"
-                render={({ field: { onChange, value, name } }) => (
-                    <Input
-                        onChange={(d: any) => {
-                            onChange(d);
-                        }}
-                        label="Page description"
-                        name={name}
-                        htmlFor={name}
-                        id={name}
-                        aria-label={'enter a description for the page'}
-                        type="text"
-                        multiline
-                        defaultValue={value}
-                    />
+                rules={maxLengthRule(2000)}
+                render={({ field: { onChange, value, name }, fieldState: { error } }) => (
+                    <>
+                        <Input
+                            onChange={(d: any) => {
+                                onChange(d);
+                            }}
```

Comment:
Issue: use of `any`, also since `onChange` is just calling the other `onChange`, i believe all you need is 
```
onChange={onChange}
```